mmdet/models/dense_heads/ascend_ssd_head.py

```diff
@@ -0,0 +1,242 @@
+            the predicted boxes and regression targets to absolute
+            coordinates format. Default False. It should be `True` when
+            using `IoULoss`, `GIoULoss`, or `DIoULoss` in the bbox head.
+        train_cfg (dict): Training config of anchor head.
+        test_cfg (dict): Testing config of anchor head.
+        init_cfg (dict or list[dict], optional): Initialization config dict.
+    """  # noqa: W605
+
+    def __init__(self,
+                 num_classes=80,
+                 in_channels=(512, 1024, 512, 256, 256, 256),
+                 stacked_convs=0,
+                 feat_channels=256,
+                 use_depthwise=False,
+                 conv_cfg=None,
+                 norm_cfg=None,
+                 act_cfg=None,
+                 anchor_generator=dict(
+                     type='SSDAnchorGenerator',
+                     scale_major=False,
+                     input_size=300,
+                     strides=[8, 16, 32, 64, 100, 300],
+                     ratios=([2], [2, 3], [2, 3], [2, 3], [2], [2]),
+                     basesize_ratio_range=(0.1, 0.9)),
+                 bbox_coder=dict(
+                     type='DeltaXYWHBBoxCoder',
+                     clip_border=True,
+                     target_means=[.0, .0, .0, .0],
+                     target_stds=[1.0, 1.0, 1.0, 1.0],
+                 ),
+                 reg_decoded_bbox=False,
+                 train_cfg=None,
+                 test_cfg=None,
+                 init_cfg=dict(
+                     type='Xavier',
+                     layer='Conv2d',
+                     distribution='uniform',
+                     bias=0)):
+        super(AscendSSDHead, self).__init__(
+            num_classes,
+            in_channels,
+            stacked_convs,
+            feat_channels,
+            use_depthwise,
+            conv_cfg,
+            norm_cfg,
+            act_cfg,
+            anchor_generator,
+            bbox_coder,
+            reg_decoded_bbox,
+            train_cfg,
+            test_cfg,
+            init_cfg)
+
+    def get_static_anchors(self, featmap_sizes, img_metas, device="cuda"):
+        if not hasattr(self, 'static_anchors') or not hasattr(self, 'static_valid_flags'):
+            static_anchors, static_valid_flags = self.get_anchors(featmap_sizes, img_metas, device)
+            self.static_anchors = static_anchors
+            self.static_valid_flags = static_valid_flags
+        return self.static_anchors, self.static_valid_flags
+
+    def get_targets(self,
+                    anchor_list,
+                    valid_flag_list,
+                    gt_bboxes_list,
+                    img_metas,
+                    gt_bboxes_ignore_list=None,
+                    gt_labels_list=None,
+                    label_channels=1,
+                    unmap_outputs=True,
+                    return_sampling_results=False,
+                    return_level=True):
+        return AscendAnchorHead.get_targets(
```

Comment:
does this mean `super().get_targets(xxx)`?

Reply:
AscendSSDHead inherit from SSDHead and AscendAnchorHead. super().get_targets(xxx) will call get_targets from SSDHead. I want call get_targets from  AscendAnchorHead